automated/linux/network-basic/network-basic.sh

```diff
@@ -68,13 +74,14 @@ case "${dist}" in
     centos|fedora)
        # dhclient exits with non-zero when it is already running.
        # Kill it first for the next dhclient test.
-       pkill dhclient || true
+       # shellcheck disable=SC2086
+       pkill ${DHCLIENT} || true
        # It takes time to kill dhclient.
        sleep 10
        ;;
 esac
 
-run "dhclient -v ${INTERFACE}" "Dynamic-Host-Configuration-Protocol-Client-dhclient-v"
+run "${DHCLIENT} ${INTERFACE}" "Dynamic-Host-Configuration-Protocol-Client-dhclient-v"
 run "route" "print-routing-tables-after-dhclient-request"
 run "ping -c 5 ${GATEWAY}" "ping-gateway"
-run "curl http://samplemedia.linaro.org/MPEG4/big_buck_bunny_480p_MPEG4_MP3_25fps_1600K_short.AVI -o ${OUTPUT}/curl_big_video.avi" "download-a-file"
+run "${CURL} http://samplemedia.linaro.org/MPEG4/big_buck_bunny_480p_MPEG4_MP3_25fps_1600K_short.AVI -o ${OUTPUT}/curl_big_video.avi" "download-a-file"
```

Comment:
`wget -o` is for output logging, not file downloading. So I'm not sure this will work with `wget` as the `${CURL}` argument here.

Reply:
Correct, should be -O. I'll try to figure out something.

Reply:
LGTM! @roxell feel free to review and merge!